lib/gui/os/window-progress.js

```diff
@@ -46,7 +47,8 @@ exports.currentWindow = electron.remote.getCurrentWindow();
 exports.set = (percentage, status) => {
   exports.currentWindow.setProgressBar(utils.percentageToFloat(percentage));
   const statusString = status === 'check' ? 'Validating' : 'Flashing';
-  exports.currentWindow.setTitle(`Etcher – ${statusString} ${percentage}%`);
+  const dash = '\u2013';
```

Comment:
It definitely works on Linux with any semi-modern DE/panel, and Windows. I think it's safe to assume it works on OS X too but I can't test this myself, @jviotti? As long as it supports unicode we're pretty much good because the dash used is in a lower and not very obscure character block, which most modern fonts support.